modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -514,13 +514,9 @@
 	new /obj/item/geiger_counter(src)
 	new /obj/item/toy/plush/mammal/fox/fuzzy(src)
 	new /obj/item/stack/marker_beacon/thirty(src)
-	new /obj/item/gun/ballistic/automatic/pistol/ninemil/ruby(src)
-	new /obj/item/ammo_box/magazine/m9mm(src)
-	new /obj/item/ammo_box/c9mm(src)
-	new /obj/item/gun/ballistic/revolver/shotpistol(src)
-	new /obj/item/ammo_box/shotgun/buck(src)
-	new /obj/item/melee/onehanded/knife/switchblade(src)
-	new /obj/item/storage/survivalkit/combat/empty(src)
+	new /obj/item/screwdriver/nuke/nt(src)
+	new /obj/item/gun_upgrade/trigger/raidertrigger(src)
+	new /obj/item/storage/survivalkit/combat(src)
```

Comment:
Whoops how did this happen. Gimme a sec I need to fix it!!

Reply:
![1691890630255839](https://github.com/ARF-SS13/coyote-bayou/assets/13924177/680eca92-a29e-4f5a-8397-7052888873b6)